src/Files.App/Helpers/Navigation/NavigationHelpers.cs

```diff
@@ -104,6 +104,21 @@ public static async Task OpenItemsWithExecutableAsync(IShellPage associatedInsta
 			}
 		}
 
+		public static async Task OpenItemsWithPythonAsync(IShellPage associatedInstance, IEnumerable<IStorageItemWithPath> items, string pythonScriptPath)
```

Comment:
@0x5bfa do you think we should create a separate helper for this?

Reply:
I think this is good